$MirOS: ports/x11/windowmaker/patches/patch-WPrefs_app_Paths_c,v 1.1.7.1 2005/03/18 16:00:58 tg Exp $
--- WPrefs.app/Paths.c.orig	Tue Oct 12 20:18:11 2004
+++ WPrefs.app/Paths.c	Wed Apr 26 12:37:03 2006
@@ -86,9 +86,8 @@ showData(_Panel *panel)
         addPathToList(panel->icoL, -1, "~/pixmaps");
         addPathToList(panel->icoL, -1, "~/GNUstep/Library/Icons");
         addPathToList(panel->icoL, -1, "/usr/include/X11/pixmaps");
-        addPathToList(panel->icoL, -1, "/usr/local/share/WindowMaker/Icons");
-        addPathToList(panel->icoL, -1, "/usr/local/share/WindowMaker/Pixmaps");
-        addPathToList(panel->icoL, -1, "/usr/share/WindowMaker/Icons");
+        addPathToList(panel->icoL, -1, "/usr/mpkg/share/WindowMaker/Icons");
+        addPathToList(panel->icoL, -1, "/usr/mpkg/share/WindowMaker/Pixmaps");
     } else {
         for (i=0; i<WMGetPropListItemCount(array); i++) {
             val = WMGetFromPLArray(array, i);
@@ -102,7 +101,7 @@ showData(_Panel *panel)
             wwarning(_("bad value in option PixmapPath. Using default path list"));
         addPathToList(panel->pixL, -1, "~/pixmaps");
         addPathToList(panel->pixL, -1, "~/GNUstep/Library/WindowMaker/Pixmaps");
-        addPathToList(panel->pixL, -1, "/usr/local/share/WindowMaker/Pixmaps");
+        addPathToList(panel->pixL, -1, "//usr/mpkgshare/WindowMaker/Pixmaps");
     } else {
         for (i=0; i<WMGetPropListItemCount(array); i++) {
             val = WMGetFromPLArray(array, i);
